src/gui/advancedsettings.cpp

```diff
@@ -418,7 +421,10 @@ void AdvancedSettings::loadAdvancedSettings()
     // Download tracker's favicon
     cb_tracker_favicon.setChecked(mainWindow->isDownloadTrackerFavicon());
     addRow(DOWNLOAD_TRACKER_FAVICON, tr("Download tracker's favicon"), &cb_tracker_favicon);
-
+    // Save path history length
+    spinDonwloadPathHistoryLength.setRange(0, INT_MAX);
+    spinDonwloadPathHistoryLength.setValue(AddNewTorrentDialog::savePathHistoryLength());
+    addRow(DOWNLOAD_PATH_HISTORY_LENGTH, tr("Download path history length"), &spinDonwloadPathHistoryLength);
```

Comment:
Is "Download path history length" a fairly clear indication of what was going on?
Although, of course, we can't have too many big names here...

Reply:
Acknowledge. Can anyone suggest a perfect short title, please?